### ISSUE TITLE
Change sys.path to include site-packages of the cmdline Python

### ISSUE THREAD

**Issue by cjrh** [cjrh/vim-conda#2]
In my vim setup, Python is fully embedded, meaning that both `pythonXY.dll` and `pythonXY.zip` (i.e. a zipped archive of the stdlib) are located in the same folder as the vim executable.  Thus, by default, inside vim, running `:py import sys; print sys.path` will **not** include the `site-packages` folder of the python that will execute my code on the command line, i.e via `:!python myfile.py`. The way jedi-vim works is to the the `sys.path` of the internal python for code completion.  Therefore, I need to modify `sys.path` in order to include the `site-packages` of the cmdline python.

This change is *already* made when changing conda envs (as per this plugin). This issue is only for the startup case:
- before a conda env change has been made, and
- the cmdline Python is different to the embedded Python, and
- the `sys.path` of the embedded Python does not include the `site-packages` folder of the cmdline Python.

**Comment by cjrh** [comment]
Another thing: I am noticing (on the current Windows machine I'm on) that the local directory is not part of the embedded python's `sys.path`. I still need to confirm whether this is just a quirk of my Windows setup (I'm using a fully-embedded Python), or whether it is also the case on other machines. I must have completion in the local folder, obviously.

[EDIT] Ok, I figured it out.  If you make Python fully embedded in vim, then `''` is not added to sys.path.  So that's a pain.  I added it manually in `vimrc`, and jedi-vim still would not complete in the current directory.  So then I added `'.'` and suddenly jedi-vim completion started working again (in the current directory; stdlib completion was working throughout.).  I don't know full what's going on yet.
